sw/device/rom_exts/manifest.md

```diff
@@ -184,13 +184,35 @@ Notes:
     This is used when signing and validating the image. This happens in the
     Mask ROM, as well as during firmware update.
 
-1.  **Usage Constraints** This is a 32-bit enumeration value which denotes what
-    kinds of devices the ROM_EXT image may be used on.
-
-    This is used by the Mask ROM to validate the image is running on the right
-    kind of hardware.
-
-    **Open Q** What possible values should this have?
+1.  **Usage Constraints** This denotes which pieces of the OpenTitan Device
+    Identifier should also be added to the digest of this image. The OpenTitan
+    Device Identifier is a 256-bit value provisioned into the device's OTP
+    memory early in manufacturing.
+
+    This allows the ROM_EXT author to constrain a ROM_EXT to a single device or
+    a group of devices, such that the ROM_EXT will not validate (and therefore
+    not boot) on devices not in that group. This can use the Silicon Creator
+    identifier, the product identifier, the individual device identifier, or any
+    data from the SKU-specific device identifier.
+
+    This is a 32-bit unsigned numeric value.
+
+    Each bit in the usage constraint value denotes a byte in the 256bit Device
```

Comment:
"This can use the Silicon Creator identifier, the product identifier, the individual device identifier" above, it says that ID can be one of 3 types, but here it talks only about the device id.

Reply:
In the "Keys and Identities" document, the whole 256-bit field that contains the vendor id, the product id, the device id, the crc, and 128 further bits, is called the "OpenTitan Device ID"

Reply:
https://docs.opentitan.org/doc/security/specs/identities_and_root_keys/#device-identifier should clarify that we mean the entire 256-bit identifier, not just the individual parts.

Reply:
The solution should not be limited to the device identifier, and the manifest should be able to handle any permutation of offsets. There are other OTP values the manufacturer may want to lock the signatures against. 

This approach is described at a high level in the ownership transfer doc, and should apply in this case as well: 

https://docs.opentitan.org/doc/security/specs/ownership_transfer/#otp_settings